docs/protos/home.md

```diff
@@ -8,10 +8,9 @@ slug: /
 
 ## Introduction
 
-Neosync, like many web services these days, is a full-fledged API. Check out the [architecture](/platform) page for an overview of how the Neosync API fits into the picture.
+Neosync has first-class APIs and SDKs that developers can use to integrate Neosync into their workflow. To learn more about how the Neosync API fits into the overall architecture, check out the Check out the [platform page](/platform).
 
-All of our APIs are generated from Protobuf files. For this reason, a client can easily be generated in any of the many supported languages.
-Internally, we make great use of the Go SDK as well as the TypeScript SDK.
+Neosync API serves up [Connect](https://github.com/connectrpc), which can listen using Connect, gRPC, or HTTP protocols. All of our APIs are generated from Protobuf files and our SDKs call gRPC endpoints. Alternatively, you can use our OpenAPI spec to generate REST clients as well.
```

Comment:
`All of our APIs are generated from Protobuf files and our SDKs call gRPC endpoints.`

Maybe a slight wording change here but they don't call the gRPC endpoints. They call the Connect endpoints by default. Each SDK can be configured to specifically call the grpc or REST endpoints however.

Moreover, not sure it's really useful info to say that our SDKs call the API endpoints. Is that not implied? (maybe it isn't but if they don't do that, then what are they doing?)

Reply:
> Alternatively, you can use our OpenAPI spec to generate REST clients as well.

We don't really have this yet (soon), so is it worth leaving out until we do?